File: cw/ruby/convert_to_binary.rb
# https://www.codewars.com/kata/59fca81a5712f9fa4700159a/train/ruby


def to_binary(n)
    bin = n.to_s(2)

    return bin.to_i
end


n = gets.to_i
print(to_binary(n))

File: cw/python/convert_to_binary.py
# https://www.codewars.com/kata/59fca81a5712f9fa4700159a/train/python


def to_binary(n):
    return int(bin(n)[2:])


if __name__ == "__main__":
    n = int(input())
    print(to_binary(n))
